packages/headless/doc-parser/tsconfig.json

```diff
@@ -1,8 +1,10 @@
 {
   "extends": "../tsconfig.json",
+  
   "compilerOptions": {
     "outDir": "build",
-    "module": "CommonJS"
+    "module": "CommonJS",
+    "lib": ["es2019"]
```

Comment:
I can revert this after this PR if you want. At some point I tried to use `flatMap` and it wasn't allowed since es2019 wasn't used.

Reply:
Could we place it in the headless root tsconfig, so that headless gets access to it too?

I think I intentionally put `build` in the file name to prevent vscode from using this file for it's type-checking, while still keeping it possible to run unit tests.